recipe/meta.yaml

```diff
@@ -33,19 +31,28 @@ requirements:
   host:
     - glib
 
-
 test:
   commands:
-    - test -f ${PREFIX}/lib/libgts.so  # [linux]
-    - test -f ${PREFIX}/lib/libgts.dylib  # [osx]
+    - test -f ${PREFIX}/lib/libgts.so                     # [linux]
+    - test -f ${PREFIX}/lib/libgts.dylib                  # [osx]
     - if not exist %LIBRARY_PREFIX%\\bin\\gts.dll exit 1  # [win]
 
 about:
   home: http://gts.sourceforge.net/
-  license: LGPL-2.0-or-later
+  license: GPL verision 2
   license_file: COPYING
-  summary: |
-    GNU Triangulated Surface Library
+  license_family: GPL2
```

Comment:
If `license` is `LGPL-2.0-or-later` so `license_family` should be `LGPL`

Reply:
Thanks, I'll fix it.